ci_scripts/build_documentation.sh

```diff
@@ -2,39 +2,46 @@
 
 script_dir="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
 
+function info {
+  echo "[$(basename ${0})] [INFO] ${1}"
```

Comment:
running `shellcheck`, this line introduces error [SC2086](https://github.com/koalaman/shellcheck/wiki/SC2086).

Looks like you need double quotes (unescaped) around the parameter to basename.
```
echo "[$(basename "${0}")] [INFO] ${1}"
```

Without the quotes, spaces in the path to the script will cause bad output.

It looks like the same mistake exists in a couple other scripts, want to fix them up too?